src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -2141,13 +2160,76 @@ private void VisitInstructionBlock(ParseNode node)
                 PushReturnList();
             
             AddFunctionJumpVars(node);
-            VisitChildNodes(node); // nextBraceIsFunction state would get incorrectly inherited by my children here if it wasn't turned off up above. 
```

Comment:
Had to explode this out so I could insert the OpcodeArgBottom at the correct index within the list where the last `DECLARE PARAMETER` statement had happened.